packages/utils/tests/resolveStaticPath.test.ts

```diff
@@ -0,0 +1,30 @@
+import { resolveStaticPath } from '../src';
```

Comment:
I am not sure if it's good idea to have this util here. I think this utils package is meant for some "generic" utils that we might need anywhere, but this util is very app specific util.

Reply:
Would make sense to move it to `suite-utils`, right?

Reply:
If yes, I will put TODO comment there

Reply:
I think so, because it's very related to Suite itself